src/TMainConsole.cpp

```diff
@@ -110,6 +110,17 @@ void TMainConsole::setLabelStyleSheet(std::string& buf, std::string& sh)
     }
 }
 
+std::optional<QString> TMainConsole::getLabelStyleSheet(const QString& name) const
+{
+    QMap<QString, TLabel*>::const_iterator it = mLabelMap.constFind(name);
+    if (it != mLabelMap.cend() && it.key() == name) {
+        return it.value()->styleSheet();
+    }
+
+    return {};
```

Comment:
Do we actually gain any performance benefit from using a `QMap<T1, T2>::constFind(const T1&)` here?
```suggestion
    return mLabelMap.value(name, std::optional<QString>());
```
Not entirely sure if that is the correct (or rather most concise) way to provide the second argument (value) if the `QMap<T1,T2>::value(const T1& key, const T2& value)` does not find the key...

Reply:
I'm not sure either... have you tested this?